src/main/groovy/com/bugsnag/android/gradle/BugsnagUploadNdkTask.groovy

```diff
@@ -152,7 +152,7 @@ class BugsnagUploadNdkTask extends BugsnagUploadAbstractTask {
                 // Call objdump, redirecting output to the output file
                 project.logger.info("${getLogPrefix()} Calling objdump process")
 
-                ProcessBuilder builder = new ProcessBuilder(objDumpPath.toString(), "--disassemble", "--demangle", "--line-numbers", "--section=.text", sharedObject.toString())
+                ProcessBuilder builder = new ProcessBuilder(objDumpPath.toString(), "-W", "-x", "--section=.debug_line", sharedObject.toString())
```

Comment:
What subset of `-W` is actually necessary (since it wraps [several options](https://sourceware.org/binutils/docs/binutils/objdump.html))? This file size could be reduced if we know specifically which components are required.